### DISCUSSION TITLE
Quote Extraction and Attribution

### DISCUSSION THREAD

**Discussion by morankyle** [explosion/spaCy#8464] [Help: Other Questions] [answered]
Apologies for doubling up on questions today, but in your experience have you seen Spacy/something particular in the Spacy universe helpful at extracting and attributing quotes from a blob of text? Only thing in this area I've seen is from StanfordNLP but its not included in Stanza

**Comment by polm** [comment, marked as answer]
I've never used it but it looks like [textacy](https://github.com/chartbeat-labs/textacy) has some kind of quote extraction. I guess [this](https://stanfordnlp.github.io/CoreNLP/quote.html) is the thing in Stanford you were referring to? 

And no worries on asking multiple questions, that's what the board is for :)

**Reply by morankyle** [reply]
Textacy does in fact have it thank you! Worked well!
